smbc-digital/iag-webapp
# feat(remove_smart): rm smart answer/result

Removal or Smart Answers/Results

Work from [Jira Card: 6626](https://stockportbi.atlassian.net/browse/DIGITAL-6226)

removed a proper job lot of files and folders... all connected with the Smart Answers/Results feature.

To test:
-) Review files removed and do a sense check
-) Pull the branch and launch ( with related feature in ContentApi )
-) Poke around and ensure all still works within the localhost
-) Run Unit tests

Will be following up with a removal of the content and content model for SmartAnswers & SmartResults within pre-prod
